Packs/CommonScriptsV2/Scripts/GetIndexOfArrayValue/GetIndexOfArrayValue.py

```diff
@@ -0,0 +1,32 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+from typing import Dict, Any
+import traceback
+
+
+
+''' MAIN FUNCTION '''
+def get_index(json_data,array_val):
+    element_index=json_data.index(array_val)
+    return element_index
+
+
+def main():
+    try:
+        json_data = argToList(demisto.args()['value'])
+        array_val = demisto.args()['array_value']
+        res=get_index(json_data,array_val)
+        demisto.results(res)
```

Comment:
Let's make sure to retrieve a full entry:
```suggestion
        demisto.results(entry = {
            'Type': entryTypes['note'],
            'Contents': {"BatchedData": {GetIndexOfArrayValue: {"array": json_data, "value": array_val, "index": res}}},
            'ContentsFormat': formats['json'],
            'HumanReadable':f"The item {array_val} can be found in index {res}.",
            'HumanReadableFormat': formats['markdown'],
        })
```

Reply:
Also, make sure to edit the outputs section in the yml and the readme accordingly.

Reply:
Updated the yml file and readme